src/lib.rs

```diff
@@ -29,7 +29,7 @@
 #![feature(ptr_metadata)]
 #![cfg_attr(feature = "exts", feature(vec_into_raw_parts))]
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
-#![no_std]
```

Comment:
Actually, I'm not sure what's best practice here. I usually go with a 

```rust
#[cfg(test)]
extern crate std;
```

as this seems to be better recognized by [IntelliJ Rust](https://intellij-rust.github.io/) - is there an official guideline?

Reply:
It turns out this change isn't needed anymore so I've reverted it. I originally added it for unit tests, but then I moved the tests into the docstring. Docstring tests act more like a separate crate, so `std` is already enabled there.

Reply:
alright :)